src/Nethermind/Nethermind.Runner/Program.cs

```diff
@@ -403,6 +426,15 @@ IConfigProvider CreateConfigProvider(ParseResult parseResult)
                 }
             }
         }
+        // For backward compatibility. To be removed in the future.
+        else if (Path.GetExtension(configFile).Equals(".cfg", StringComparison.Ordinal))
+        {
+            var name = Path.GetFileNameWithoutExtension(configFile)!;
+
+            configFile = $"{configFile[..^4]}.json";
```

Comment:
```suggestion
            configFile = $"{name}.json";
```

Reply:
@LukaszRozmej At this point, `configFile` is the full path, while `name` is just the file name. So they're different.